app/src/org/commcare/android/database/connect/models/ConnectLinkedAppRecord.java

```diff
@@ -11,11 +12,16 @@ public class ConnectLinkedAppRecord extends Persisted {
      */
     public static final String STORAGE_KEY = "app_info";
 
+    public static final String META_APP_ID = "app_id";
+    public static final String META_USER_ID = "user_id";
+
     @Persisting(1)
-    private String appID;
+    @MetaField(META_APP_ID)
+    private String app_id;
```

Comment:
why change is to `_`  ? We should keep them camelcased as before if possible as per Java conventions.

Reply:
Done